deps/build.jl

```diff
@@ -7,6 +7,8 @@ using BinDeps, Compat
     provides(AptGet, "hdf5-tools", hdf5)
     provides(Pacman, "hdf5", hdf5)
     provides(Yum, "hdf5", hdf5)
+    provides(Sources, URI("http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-1.8.16.tar.gz"), hdf5)
+    provides(BuildProcess, Autotools(libtarget = "libhdf5.la"), hdf5)
```

Comment:
That's true, but these lines require a `hdf5 = library_dependency(...)` call before them, so they cannot exist stand-alone. It would be possible to have a fourth, generic section below the three OS-specific sections. However, since both Windows and OS X already have other providers that should be faster, since they download a binary instead of installing from source, this fourth section would only be used for Linux.

I can still make that change, of course.